tests/components/config/test_core.py

```diff
@@ -135,26 +143,28 @@
 async def test_detect_config_fail(hass, client):
     """Test detect config."""
     with patch(
-        "homeassistant.util.location.async_detect_location_info",
-        return_value=mock_coro(
-            location.LocationInfo(
-                ip=None,
-                country_code=None,
-                country_name=None,
-                region_code=None,
-                region_name=None,
-                city=None,
-                zip_code=None,
-                latitude=None,
-                longitude=None,
-                use_metric=True,
-                time_zone="Europe/Amsterdam",
-            )
-        ),
+            "homeassistant.util.location.async_detect_location_info",
+            return_value=mock_coro(
+                location.LocationInfo(
+                    ip=None,
+                    country_code=None,
+                    country_name=None,
+                    region_code=None,
+                    region_name=None,
+                    city=None,
+                    zip_code=None,
+                    latitude=None,
+                    longitude=None,
+                    use_metric=True,
+                    time_zone="Europe/Amsterdam",
+                )),
     ):
         await client.send_json({"id": 1, "type": "config/core/detect"})
 
         msg = await client.receive_json()
 
     assert msg["success"] is True
-    assert msg["result"] == {"unit_system": "metric", "time_zone": "Europe/Amsterdam"}
+    assert msg["result"] == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)